ui/components/component-library/text/text.scss

```
@@ -26,6 +33,21 @@ $text-variants: (
   }
 }
 
+// Brand evolution font mixin
+@mixin brand-evolution-font($type) {
+  $font-family: map-get($brand-evolution-font-families, $type);
+
+  @if $font-family {
+    font-family: $font-family;
+    background: green;
+  }
+
+  @else {
+    font-family: var(--font-family-sans); // Fallback to default sans font if type not found
+    background: red;
+  }
```

```suggestion
    font-family: $font-family, var(--font-family-sans);
```

This should likely work